examples/app-service-asp-net/variables.tf

```diff
@@ -0,0 +1,45 @@
+variable "groupName" {
```

Comment:
_minor_ I might suggest renaming this `resource_group_name` to be clearer (and underscores for consistency) -  (I'll push a commit for this)